src/mmrelay/meshtastic_utils.py

```
@@ -4156,30 +4185,35 @@ def on_meshtastic_message(packet: dict[str, Any], interface: Any) -> None:
             - optional 'channel' (mapped channel value)
         interface: Meshtastic interface used to resolve node information and the relay node id. Must provide .myInfo.my_node_num and a .nodes mapping for sender metadata.
     """
-    global config, matrix_rooms
+    global config, matrix_rooms, _relay_rx_time_clock_skew_secs
```

Ruff (0.15.7)

[warning] Using global for `config` but no assignment is done

(PLW0602)